Backpack/src/main/java/net/skyscanner/backpack/calendar2/list/CalendarCellHeaderHolder.kt

```diff
@@ -48,8 +49,17 @@ internal class CalendarCellHeaderHolder(
   override fun bind(model: CalendarCell.Header) {
     month.text = model.title
     btnSelectWholeMonth.apply {
-      model.selectWholeMonthLabel?.let { text = it }
-      isVisible = !model.selectWholeMonthLabel.isNullOrEmpty()
+      when (model.monthSelectionMode) {
+        CalendarParams.MonthSelectionMode.Disabled -> {
+          isEnabled = false
+          isVisible = false
+        }
+        is CalendarParams.MonthSelectionMode.SelectWholeMonth -> {
+          isEnabled = true
```

Comment:
I think there's no need to disable it if we set it to View.GONE. But it needs to be disabled when calendar is disabled and MonthSelectionMode == SelectWholeMonth.

Reply:
Well spotted! Just added [another commit](https://github.com/Skyscanner/backpack-android/pull/1110/commits/2a3c5983d60f31dab0c19b1415c7cca2a1724aee) fixing this